ENIGMAsystem/SHELL/Graphics_Systems/OpenGL1/OPENGLStd.cpp

```diff
@@ -36,29 +36,12 @@ namespace enigma
   void graphicssystem_initialize()
   {
     //enigma::pbo_isgo=GL_ARB_pixel_buffer_object;
-    glMatrixMode(GL_PROJECTION);
-    glClearColor(0,0,0,0);
-    glMatrixMode(GL_MODELVIEW);
-    glLoadIdentity();
-
-    using enigma_user::room_width;
-    using enigma_user::room_height;
-    glViewport(0,0,(int)room_width,(int)room_height);
-    glOrtho(-1,(int)room_width,-1,(int)room_height,0,1);
-    glClear (GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
-
-    glDisable(GL_DEPTH_TEST);
-    glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
 
     glEnable(GL_BLEND);
-    glEnable(GL_ALPHA_TEST);
```

Comment:
I would leave this here until the other routines are also re-homed. I don't mind encapsulating these more granularly, but I think `graphicssystem_initialize()` is a pretty good encapsulation, as-is.